packages/github_cli/project.bri

```diff
@@ -5,7 +5,9 @@ import { goBuild } from "go";
 export const project = {
   name: "github_cli",
   version: "2.72.0",
-  latestBuildDate: "2025-04-30",
+  extra: {
+    buildDate: "20241220221126",
```

Comment:
Looks like this format differs from what the `live-update` script would write

Reply:
I fixed it.